chia/rpc/wallet_rpc_client.py

```diff
@@ -586,11 +586,11 @@ async def add_uri_to_nft(self, wallet_id, nft_coin_id, uri, fee):
         response = await self.fetch("nft_add_uri", request)
         return response
 
-    async def transfer_nft(self, wallet_id, nft_coin_id, artist_address, fee):
+    async def transfer_nft(self, wallet_id, nft_coin_id, target_address, fee):
```

Comment:
This change doesn't seem relevant to the RPC addition. Was this included on purpose?

Reply:
I think the old name of the variable is confusing, transfer to the artist doesn't looks right to me. The front-end may need some change but I think it's worth to change.